### ISSUE TITLE
Mypy v0.8 support

### ISSUE THREAD

**Issue by sondrelg** [typeddjango/django-stubs#565]
Hi,

I tried looking through recent issues and PRs but was unable to find anything related to this:

Are there any near term plans to release a new `django-stubs` version with support for Mypy v0.800? 

I'm sure the latest `django-stubs` release already works fairly well with this, but we use [Poetry](https://python-poetry.org/) for dependency management, and the Poetry resolver does not let you have conflicting dependencies:

```
  SolverProblemError

  Because no versions of django-stubs match >1.7.0,<2.0.0
   and django-stubs (1.7.0) depends on mypy (>=0.790), django-stubs (>=1.7.0,<2.0.0) requires mypy (>=0.790).
  So, because cloud depends on both mypy (^0.8.000) and django-stubs (^1.7.0), version solving failed.
```

In other words, the `mypy>=0.790` dependency specification [here](https://github.com/typeddjango/django-stubs/blob/master/setup.py) is blocking us from adopting the newest version. 

There also does not seem to be a good way of getting around this (see [this issue](https://github.com/python-poetry/poetry/issues/697)), currently.

What, if anything, needs to be done, to release a new version?

**Comment by eabruzzese** [comment]
@sondrelg I'm using Poetry for dependency management as well, and don't have any issues using the latest `mypy` (0.812) with the latest `django-stubs` (1.7.0).

Here's a snippet of my `pyproject.toml`:

```toml
[tool.poetry.dev-dependencies]
mypy = "^0.812"
mypy-extensions = "^0.4.3"
django-stubs = "^1.7.0"
```

And output from mypy working as expected:

```
[development/local /app] mypy --version
mypy 0.812
[development/local /app] mypy api/models/constants.py
Success: no issues found in 1 source file
```

One thing I noticed from your pasted output: `depends on both mypy (^0.8.000) ...` -- are you specifying `^0.8`? Perhaps it would work if you try `^0.812`?

**Comment by sondrelg** [comment]
That is very possible! I'll check and report back tomorrow morning; thanks!

**Comment by sondrelg** [comment]
I re-ran the update today and it works.. without changing anything 🤨  Looking at the error I posted, I agree I must have put in the `mypy` versions wrong initially. 

In past Sondre's defence, the mypy versioning is very confusing when 99% of packages are using semver  😅

Thank you @eabruzzese! 👏
